### PR TITLE
PTL-1667 - adding interop notifications listener doc

### PR DESCRIPTION
**Pull request by patrickoneill-genesis**
ATTENTION! NEW DOCS STRUCTURE
FOR THE NEXT THREE WEEKS, ONLY BRANCH FROM new-docs-structure
OTHERWISE, WAIT UNTIL THE NEW STRUCTURE HAS BEEN RELEASED

  - Please check the [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4) for details
  
__________

Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?

Have you done a trial build to check all new or changed links?

Is there anything else you would like us to know?

**This week's exciting advice from the style guide**

- Write your headings in sentence case:

  - A good example
    This is a correct heading.
  - A Bad Example
    This is not a correct heading.

### CODE REVIEW

File: docs/001_develop/03_client-capabilities/021_desktop-interoperability/006_interop_notifications_listener.mdx [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+<TabItem value="react">
+Declaration
+```html
+<rapid-interop-notification-listener></rapid-interop-notification-listener>
+```
+
+Usage
+```typescript
+export function MyComponent() {
```
Comment (matteematt) [inline docs/001_develop/03_client-capabilities/021_desktop-interoperability/006_interop_notifications_listener.mdx:69]: In react 19 (which we're targetting) you should be able to set `toastButtons` directly. However, if you want to keep it like this you're missing the `useRef` decleration

File: docs/001_develop/03_client-capabilities/021_desktop-interoperability/006_interop_notifications_listener.mdx [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+<rapid-interop-notification-listener></rapid-interop-notification-listener>
+```
+
+Usage
+```typescript
+import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+@Component({
+    selector: 'my-root',
+    templateUrl: './my.component.html',
+    template: `
```
Comment (matteematt) [inline docs/001_develop/03_client-capabilities/021_desktop-interoperability/006_interop_notifications_listener.mdx:108]: I assume you don't need `templateUrl` in this case?

File: docs/001_develop/03_client-capabilities/021_desktop-interoperability/006_interop_notifications_listener.mdx [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+        <tr>
+            <td>toastButtons</td>
+<td>
+ToastButton[] <!-- To Do add link-->
+</td>
+            <td>An array of configs for showing buttons. Buttons appear in notifications when the <code>condition</code> callback returns <code>true</code>. The action callback is called when the corresponding button is clicked</td>
+<td>
+
+```typescript
+    <rapid-interop-notification-listener :toastButtons=${)_ => [
```
Comment (matteematt) [inline docs/001_develop/03_client-capabilities/021_desktop-interoperability/006_interop_notifications_listener.mdx:192]:
```suggestion
    <rapid-interop-notification-listener :toastButtons=${(_ )=> [

```

**Review by matteematt — COMMENTED** [review]
Nice one, just a few small bits. Btw don't forget to add the `new-doc-structure` tag on your github PR so we ensure these are visible for review

File: docs/001_develop/03_client-capabilities/021_desktop-interoperability/006_interop_notifications_listener.mdx (hunk 2 above)
Comment (patrickoneill-genesis) [reply docs/001_develop/03_client-capabilities/021_desktop-interoperability/006_interop_notifications_listener.mdx:108]: Removed

File: docs/001_develop/03_client-capabilities/021_desktop-interoperability/006_interop_notifications_listener.mdx (hunk 3 above)
Comment (patrickoneill-genesis) [reply docs/001_develop/03_client-capabilities/021_desktop-interoperability/006_interop_notifications_listener.mdx:192]: Fixed

File: docs/001_develop/03_client-capabilities/021_desktop-interoperability/006_interop_notifications_listener.mdx (hunk 1 above)
Comment (patrickoneill-genesis) [reply docs/001_develop/03_client-capabilities/021_desktop-interoperability/006_interop_notifications_listener.mdx:69]: Sticking with useRef for now as that's what I've tested

**Review by matteematt — APPROVED** [review]

**Pull request merged**